spiceaidocs/docs/cli/reference/dataset.md

```diff
@@ -15,8 +15,24 @@ spice dataset [command]
 
 Available `command`s:
 
-- `configure`:    Configure a dataset
+- `configure`: Create/configure a dataset directly from the command-line, including customizing components such as remote source (`from` in a Spicepod) and acceleration along with metadata such as name and description.
```

Comment:
This should be a simple one line summary here. Detailed summary should be in the actual command documentation.

Reply:
I'm a bit confused here. The file you're referencing is the primary documentation for the `spice dataset` command, so shouldn't `spice dataset configure` be described in detail here? Let me know if I'm misunderstanding something.